#!/bin/sh
patch -p1 -f $* < $0
exit $?

From 56ed4e3c082bbf349b51a873cf09a4b1ed0ec5ef Mon Sep 17 00:00:00 2001
From: Thorsten Maerz <info@netztorte.de>
Date: Fri, 30 Apr 2021 15:44:25 +0200
Subject: [PATCH] Allow compilation from outside source directory

The autogenerated files (matcher_parser_*, quote_fmt_*) are created in
builddir (not in srcdir).
Updated IFLAGS (include paths), so these files are found.
---
 src/plugins/address_keeper/Makefile.am | 1 +
 src/plugins/att_remover/Makefile.am    | 1 +
 src/plugins/attachwarner/Makefile.am   | 1 +
 src/plugins/gdata/Makefile.am          | 1 +
 src/plugins/libravatar/Makefile.am     | 1 +
 src/plugins/managesieve/Makefile.am    | 1 +
 src/plugins/pdf_viewer/Makefile.am     | 1 +
 src/plugins/pgpcore/Makefile.am        | 1 +
 src/plugins/pgpmime/Makefile.am        | 1 +
 src/plugins/smime/Makefile.am          | 1 +
 src/plugins/vcalendar/Makefile.am      | 1 +
 11 files changed, 11 insertions(+)

diff --git a/src/plugins/address_keeper/Makefile.am b/src/plugins/address_keeper/Makefile.am
index 2388f7759..180f9233c 100644
--- a/src/plugins/address_keeper/Makefile.am
+++ b/src/plugins/address_keeper/Makefile.am
@@ -6,6 +6,7 @@
 EXTRA_DIST = claws.def plugin.def version.rc
 
 IFLAGS = \
+	-I$(top_builddir)/src \
 	-I$(top_srcdir)/src \
 	-I$(top_srcdir)/src/common \
 	-I$(top_builddir)/src/common \
diff --git a/src/plugins/att_remover/Makefile.am b/src/plugins/att_remover/Makefile.am
index a0f8b9232..4bf90a6c2 100644
--- a/src/plugins/att_remover/Makefile.am
+++ b/src/plugins/att_remover/Makefile.am
@@ -6,6 +6,7 @@
 EXTRA_DIST = claws.def plugin.def version.rc
 
 IFLAGS = \
+	-I$(top_builddir)/src \
 	-I$(top_srcdir)/src \
 	-I$(top_srcdir)/src/common \
 	-I$(top_builddir)/src/common \
diff --git a/src/plugins/attachwarner/Makefile.am b/src/plugins/attachwarner/Makefile.am
index 6315e9165..5f06df79e 100644
--- a/src/plugins/attachwarner/Makefile.am
+++ b/src/plugins/attachwarner/Makefile.am
@@ -6,6 +6,7 @@
 EXTRA_DIST = claws.def plugin.def version.rc
 
 IFLAGS = \
+	-I$(top_builddir)/src \
 	-I$(top_srcdir)/src \
 	-I$(top_srcdir)/src/common \
 	-I$(top_builddir)/src/common \
diff --git a/src/plugins/gdata/Makefile.am b/src/plugins/gdata/Makefile.am
index 39db98c0c..511dc6b65 100644
--- a/src/plugins/gdata/Makefile.am
+++ b/src/plugins/gdata/Makefile.am
@@ -23,6 +23,7 @@ gdata_la_LDFLAGS = \
 	$(GDATA_LIBS)
 
 gdata_la_CPPFLAGS = \
+	-I$(top_builddir)/src \
 	-I$(top_srcdir)/src \
 	-I$(top_srcdir)/src/common \
 	-I$(top_builddir)/src/common \
diff --git a/src/plugins/libravatar/Makefile.am b/src/plugins/libravatar/Makefile.am
index c494b039d..704fc9d29 100644
--- a/src/plugins/libravatar/Makefile.am
+++ b/src/plugins/libravatar/Makefile.am
@@ -6,6 +6,7 @@
 EXTRA_DIST = claws.def plugin.def version.rc
 
 IFLAGS = \
+	-I$(top_builddir)/src \
 	-I$(top_srcdir)/src \
 	-I$(top_srcdir)/src/common \
 	-I$(top_builddir)/src/common \
diff --git a/src/plugins/managesieve/Makefile.am b/src/plugins/managesieve/Makefile.am
index fe4cb7d31..ad3f97626 100644
--- a/src/plugins/managesieve/Makefile.am
+++ b/src/plugins/managesieve/Makefile.am
@@ -6,6 +6,7 @@
 EXTRA_DIST = claws.def plugin.def version.rc
 
 IFLAGS = \
+	-I$(top_builddir)/src \
 	-I$(top_srcdir)/src \
 	-I$(top_srcdir)/src/common \
 	-I$(top_builddir)/src/common \
diff --git a/src/plugins/pdf_viewer/Makefile.am b/src/plugins/pdf_viewer/Makefile.am
index c790e9426..d5ce4b22e 100644
--- a/src/plugins/pdf_viewer/Makefile.am
+++ b/src/plugins/pdf_viewer/Makefile.am
@@ -19,6 +19,7 @@ pdf_viewer_la_LDFLAGS = \
 	$(POPPLER_LIBS)
 
 pdf_viewer_la_CPPFLAGS = \
+	-I$(top_builddir)/src \
 	-I$(top_srcdir)/src \
 	-I$(top_srcdir)/src/common \
 	-I$(top_builddir)/src/common \
diff --git a/src/plugins/pgpcore/Makefile.am b/src/plugins/pgpcore/Makefile.am
index 1c4046752..e1275023e 100644
--- a/src/plugins/pgpcore/Makefile.am
+++ b/src/plugins/pgpcore/Makefile.am
@@ -11,6 +11,7 @@ endif
 EXTRA_DIST = version.rc plugin.def claws.def
 
 IFLAGS = \
+	-I$(top_builddir)/src \
 	-I$(top_srcdir)/src \
 	-I$(top_srcdir)/src/common \
 	-I$(top_builddir)/src/common \
diff --git a/src/plugins/pgpmime/Makefile.am b/src/plugins/pgpmime/Makefile.am
index 167894fb1..67a87a9a1 100644
--- a/src/plugins/pgpmime/Makefile.am
+++ b/src/plugins/pgpmime/Makefile.am
@@ -6,6 +6,7 @@
 EXTRA_DIST = version.rc plugin.def claws.def mypgpcore.def pgpmime.deps
 
 IFLAGS = \
+	-I$(top_builddir)/src \
 	-I$(top_srcdir)/src \
 	-I$(top_srcdir)/src/common \
 	-I$(top_builddir)/src/common \
diff --git a/src/plugins/smime/Makefile.am b/src/plugins/smime/Makefile.am
index d39423337..2d25f6e65 100644
--- a/src/plugins/smime/Makefile.am
+++ b/src/plugins/smime/Makefile.am
@@ -6,6 +6,7 @@
 EXTRA_DIST = version.rc plugin.def claws.def mypgpcore.def smime.deps
 
 IFLAGS = \
+	-I$(top_builddir)/src \
 	-I$(top_srcdir)/src \
 	-I$(top_srcdir)/src/common \
 	-I$(top_builddir)/src/common \
diff --git a/src/plugins/vcalendar/Makefile.am b/src/plugins/vcalendar/Makefile.am
index 8bceff9e6..cdd570ccf 100644
--- a/src/plugins/vcalendar/Makefile.am
+++ b/src/plugins/vcalendar/Makefile.am
@@ -6,6 +6,7 @@
 EXTRA_DIST = claws.def plugin.def version.rc
 
 IFLAGS = \
+	-I$(top_builddir)/src \
 	-I$(top_srcdir)/src \
 	-I$(top_srcdir)/src/common \
 	-I$(top_builddir)/src/common \
-- 
2.25.1

